### PR TITLE
[14.0][IMP] fieldservice: add fsm_location as res.partner type

### PR DESCRIPTION
**Pull request by hparfr**
replace #696

### CODE REVIEW

**Comment by hparfr** [comment]
ping @mourad-ehm

**Review by mourad-ehm — APPROVED** [review]

File: fieldservice/models/res_partner.py
Comment (patrickrwilson) [inline fieldservice/models/res_partner.py:10]: 14.0 Requires ondelete= attribute. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Just one small item, LGTM after change.

**Review by mourad-ehm — APPROVED** [review]

**Comment by chienandalu** [comment]
The merge command didn't have effect

**Pull request merged**
